File: src/main/java/io/bindernews/thegrackle/relics/PhoenixIdol.java
package io.bindernews.thegrackle.relics;

import io.bindernews.thegrackle.GrackleMod;

public class PhoenixIdol extends BaseRelic {
    public static final String ID = GrackleMod.makeId("PhoenixIdol");

    public PhoenixIdol() {
        super(ID, RelicTier.RARE, LandingSound.MAGICAL);
    }

    @Override
    public void onLoseHp(int damageAmount) {
        super.onLoseHp(damageAmount);
    }
}

File: src/main/kotlin/io/bindernews/thegrackle/relics/PhoenixIdol.kt
package io.bindernews.thegrackle.relics

import basemod.abstracts.CustomRelic
import io.bindernews.thegrackle.helper.makeId

class PhoenixIdol : CustomRelic(ID, "", RelicTier.RARE, LandingSound.MAGICAL) {
    companion object {
        @JvmField val ID = makeId(PhoenixIdol::class)
    }
}